pkg/testing/integration/command.go

```diff
@@ -56,10 +58,13 @@ func RunCommand(t *testing.T, name string, args []string, wd string, opts *Progr
 		cmd.Stdout = opts.Stdout
 		cmd.Stderr = opts.Stderr
 		runerr = cmd.Run()
+
 	} else {
 		runout, runerr = cmd.CombinedOutput()
 	}
 
+	defer orphanCleaner.killOrphanProcesses(t)
```

Comment:
We might already have code that does what you need: https://github.com/pulumi/pulumi/blob/master/sdk/go/common/util/cmdutil/child_windows.go#L26-L73

Reply:
Nice! Yes this is pretty much the same idea modulo logging, let me reuse.